sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/concurrent/JreDoubleAdder.java

```diff
@@ -5,6 +5,9 @@
 
 package io.opentelemetry.sdk.metrics.internal.concurrent;
 
+import org.codehaus.mojo.animal_sniffer.IgnoreJRERequirement;
+
+@IgnoreJRERequirement
```

Comment:
This annotation unfortunately isn't doing anything (we had to exclude the class entirely). We could still add it but I guess better to remove from JreLongAdder which I left in accidentally I think.

Reply:
Ah I see. I knew it didn't do anything but thought you had added it to `JreLongAdder` just as a reminder to future maintainers. 

Happy to remove it from both places.

Reply:
Keeping in both as a reminder makes sense to me, I'm OK with either. To clarify, I had no lofty ideas in mind when keeping in JreLongAdder :P